aptos-move/framework/src/release_builder.rs

```diff
@@ -66,21 +80,30 @@ impl ReleaseOptions {
             source_paths.push(relative_path.display().to_string());
         }
         let bundle = ReleaseBundle::new(released_packages, source_paths);
-        std::fs::create_dir_all(output.parent().unwrap())?;
-        std::fs::write(&output, bcs::to_bytes(&bundle)?)?;
+        let parent = output.parent().expect("Failed to get parent directory");
+        std::fs::create_dir_all(parent).context("Failed to create dirs")?;
+        std::fs::write(&output, bcs::to_bytes(&bundle)?).context("Failed to write output")?;
         Ok(())
     }
 
     fn generate_rust_bindings(abis: &[EntryABI], path: &Path) -> anyhow::Result<()> {
         {
-            let mut file = std::fs::File::create(path)?;
-            rust::output(&mut file, abis, true)?;
+            let mut file = std::fs::File::create(path)
+                .with_context(|| format!("Failed to create {}", path.display()))?;
+            rust::output(&mut file, abis, true)
+                .with_context(|| format!("Failed to output rust bindings to {}", path.display()))?;
         }
         std::process::Command::new("rustfmt")
             .arg("--config")
             .arg("imports_granularity=crate")
             .arg(path)
-            .status()?;
+            .output()
+            .with_context(|| {
+                format!(
+                    "Failed to run rustfmt on {}, is rustfmt installed?",
```

Comment:
This is the important part, the build was failing because it relies on rustfmt being present.